TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/TeleOp.java

```diff
@@ -31,6 +32,13 @@ public class TeleOp extends OpMode {
     private final double jointTargetPos = 0.0;
     boolean scoringATM;
 
+    Gamepad currentGamepad1 = new Gamepad();
+    Gamepad currentGamepad2 = new Gamepad();
+
+    Gamepad previousGamepad1  = new Gamepad();
+    Gamepad previousGamepad2  = new Gamepad();
```

Comment:
okay i see what you're doing with current gamepad and previous gamepad but it really just feels like you're implying that we're physically switching them out. is there any way you could be more clear in your naming?  like maybe clarify that previous gamepads are just records of past states rather than an actual physical gamepad. if not just add a clarifying comment when you initialize them.